tests/test_wave_03.py

```diff
@@ -43,24 +43,24 @@ def test_friends_unique_movies():
     assert FANTASY_4 in friends_unique_movies
     assert amandas_data == clean_wave_3_data()
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
 def test_friends_unique_movies_not_duplicated():
     # Arrange
     amandas_data = clean_wave_3_data()
     amandas_data["friends"][0]["watched"].append(INTRIGUE_3)
 
     # Act
     friends_unique_movies = get_friends_unique_watched(amandas_data)
+    intrigue_count = friends_unique_movies.count(INTRIGUE_3)
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
Nicely done! These assertions work great, especially when you don't know the list order! In this case, we do, so the following also works:
```python
assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
```